lib/PuppeteerSharp/Launcher.cs

```diff
@@ -100,6 +111,25 @@ public async Task<Browser> ConnectAsync(ConnectOptions options)
             }
         }
 
+        private async Task<string> GetWSEndpointAsync(string browserURL)
+        {
+            try
+            {
+                if (Uri.TryCreate(new Uri(browserURL), "/json/version", out var endpointURL))
+                {
+                    var client = new HttpClient();
```

Comment:
shouldn't we create a static `HttpClient` or at least a single instance for the browser?

Reply:
I don’t think this part is critical. When do you hit Launcher.ConnectAsync will be call too many time so as a local HttpClient instance would be a problem?

Reply:
I don’t think this part is critical. When do you hit Launcher.ConnectAsync will be call too many time so as a local HttpClient instance would be a problem?

Reply:
OK, I wasn't sure about this but just wanted to be sure